research/2d-decoding/rectangle_detector.py

```diff
@@ -157,33 +173,36 @@ def readImage(filename):
             continue
 
         try:
-            line = lambda a: (minX[1]-minY[1])/(minX[0]-minY[0])*(a - minX[0]) + minX[1] # Upper left side
+            def line(a): return (
```

Comment:
See above

Reply:
Outdated + couldn't find the code in my codebase